taxonomies/ingredients.txt

```diff
@@ -32291,7 +32313,7 @@ it:frutta secca
 
 en:juice
 de:Saft
-es:Zumo
+es:Zumo, nectar
```

Comment:
nectar should have its own entry, as it's different from juice (nectar is juice + water + sugar)

Reply:
ok, removed